src/baltpeter/Internetmarke/StampPngResult.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace baltpeter\Internetmarke;
+
+class StampPngResult extends ApiResult {
+  /**
+   * @var string url to zipfile
+   */
+  public $link;
+  /**
+   * @var integer portokasse balance in eurocent
+   */
+  public $walletBallance;
+  /**
+   * @var stdClass with attributes `shopOrderId` and `voucherList`
+   */
+  public $shoppingCart;
+
+  /**
+   *
+   * @param string $link           url to zipfile
+   * @param integer $walletBallance portokasse balance in eurocent
+   * @param stdClass $shoppingCart   with attributes `shopOrderId` and `voucherList`
+   */
+  public function __construct($link, $walletBallance, $shoppingCart) {
+      $this->link           = $link;
+      $this->walletBallance = $walletBallance;
+      $this->shoppingCart   = $shoppingCart;
+  }
```

Comment:
Aren't we losing the `manifestLink` here? That should also be included.

Reply:
Good catch